aspnet/conceptual-overview/understanding-aspnet5-apps.rst

```diff
@@ -36,192 +29,142 @@ has been refactored into files that are generally smaller and more focused.
+The file based approach presented several problems. First, protecting sensitive project files required framework-level protection of certain filenames or extensions, to prevent having things like *web.config* or *global.asax* served to a client in response to a request. Having to specifically block access (also known as blacklisting) to certain files is much less secure than granting access only to those files which should be accessible (also known as whitelisting). Typically, different versions were required for dev/test and production (for example *web.config*).  Scripts would typically be referenced individually and in a readable format during development, but would be `bundled and minified <https://docs.asp.net/en/latest/client-side/bundling-and-minification.html>`_ together when deployed for production. It’s beneficial to deploy only production files to production, but handling these kinds of scenarios was difficult with the previous file structure.
 
 Enter the *wwwroot* folder in ASP.NET 5. The *wwwroot* folder represents the actual root of the web app when running on a web server. Static files, like *appsettings.json*, which are not located in *wwwroot* will never be accessible, and there is no need to create special rules to block access to sensitive files. Instead of blacklisting access to sensitive files, a more secure whitelist approach is taken whereby only those files in the *wwwroot* folder are accessible via web requests.
 
-In addition to the security benefits, the *wwwroot* folder also simplifies common tasks like bundling and minification, which can now be more easily incorporated into a standard build process and automated using tools like Grunt.
+In addition to the security benefits, the *wwwroot* folder also simplifies common tasks like bundling and minification, which can now be more easily incorporated into a standard build process and automated using tools like `Gulp <https://docs.asp.net/en/latest/client-side/using-gulp.html>`_ and `Grunt <https://docs.asp.net/en/latest/client-side/using-grunt.html>`_.
 
 .. _`client side dependencies` :
 
 Client Side Dependency Management
 ---------------------------------
 
-The Dependencies folder contains two subfolders: Bower and NPM. These folders correspond to two package managers by the same names, and they’re used to pull in client-side dependencies and tools (e.g. jQuery, bootstrap, or grunt). Expanding the folders reveals which dependencies are currently managed by each tool, and the current version being used by the project.
+The *Dependencies* folder contains two subfolders: *Bower* and *NPM*. These folders correspond to two package managers by the same names, and they’re used to pull in client-side dependencies and tools (e.g. jQuery, bootstrap, or gulp). Expanding the folders reveals which dependencies are currently managed by each tool, and the current version being used by the project.
```

Comment:
I think Bootstram and Gulp should capitalized in this case. Can we also add links to the corresponding project pages in case folks don't know what they are?